components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/service/api/admin/UserManagementAdminService.java

```diff
@@ -72,7 +72,7 @@
         "Further, this is strictly restricted to admin users only ")
 public interface UserManagementAdminService {
 
-    @PUT
+    @POST
```

Comment:
Any specific reason to change this into POST? POST should used in places for creating resource or processing function resource(Please refer [WSO2 REST API Guideline](http://wso2.com/whitepapers/wso2-rest-apis-design-guidelines/#73)). PUT was most appropriate here since we updating user's credentials resource.